lexers/todotxt.lua

```diff
@@ -0,0 +1,141 @@
+local priority_e = lex:tag(lexer.LIST .. '.priority.e', lexer.starts_line('(E) '))
+local priority_f = lex:tag(lexer.LIST .. '.priority.f', lexer.starts_line('(F) '))
+-- See priority_g_z
+--local priority_g = lex:tag(lexer.LIST .. '.priority.g', lexer.starts_line('(G) '))
+--local priority_h = lex:tag(lexer.LIST .. '.priority.h', lexer.starts_line('(H) '))
+--local priority_i = lex:tag(lexer.LIST .. '.priority.i', lexer.starts_line('(I) '))
+--local priority_j = lex:tag(lexer.LIST .. '.priority.j', lexer.starts_line('(J) '))
+--local priority_k = lex:tag(lexer.LIST .. '.priority.k', lexer.starts_line('(K) '))
+--local priority_l = lex:tag(lexer.LIST .. '.priority.l', lexer.starts_line('(L) '))
+--local priority_m = lex:tag(lexer.LIST .. '.priority.m', lexer.starts_line('(M) '))
+--local priority_n = lex:tag(lexer.LIST .. '.priority.n', lexer.starts_line('(N) '))
+--local priority_o = lex:tag(lexer.LIST .. '.priority.o', lexer.starts_line('(O) '))
+--local priority_p = lex:tag(lexer.LIST .. '.priority.p', lexer.starts_line('(P) '))
+--local priority_q = lex:tag(lexer.LIST .. '.priority.q', lexer.starts_line('(Q) '))
+--local priority_r = lex:tag(lexer.LIST .. '.priority.r', lexer.starts_line('(R) '))
+--local priority_s = lex:tag(lexer.LIST .. '.priority.s', lexer.starts_line('(S) '))
+--local priority_t = lex:tag(lexer.LIST .. '.priority.t', lexer.starts_line('(T) '))
+--local priority_u = lex:tag(lexer.LIST .. '.priority.u', lexer.starts_line('(U) '))
+--local priority_v = lex:tag(lexer.LIST .. '.priority.v', lexer.starts_line('(V) '))
+--local priority_w = lex:tag(lexer.LIST .. '.priority.w', lexer.starts_line('(W) '))
+--local priority_x = lex:tag(lexer.LIST .. '.priority.x', lexer.starts_line('(X) '))
+--local priority_y = lex:tag(lexer.LIST .. '.priority.y', lexer.starts_line('(Y) '))
+--local priority_z = lex:tag(lexer.LIST .. '.priority.z', lexer.starts_line('(Z) '))
+
+local priority_g_z = lex:tag(lexer.LIST .. '.priority.g_z', lexer.starts_line(P('(') * lexer.upper * P(') ')))
+
+-- priority A-F can have different styles, G-Z all the same - similar approach as Markor Android app https://github.com/gsantner/markor/blob/master/app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtBasicSyntaxHighlighter.java#L13
+-- in theme properties:
+--      scintillua.styles.list.priority.a
+--      scintillua.styles.list.priority.b
+--      scintillua.styles.list.priority....
+-- Example
+--      scintillua.styles.list.priority.a=fore:$(scintillua.colors.red),bold
+--      scintillua.styles.list.priority.b=fore:$(scintillua.colors.orange),bold
+--      scintillua.styles.list.priority.c=fore:$(scintillua.colors.green),bold
+--      scintillua.styles.list.priority.d=fore:$(scintillua.colors.blue),bold
+--      scintillua.styles.list.priority.e=fore:$(scintillua.colors.purple),bold
+--      scintillua.styles.list.priority.f=fore:$(scintillua.colors.dark_grey),bold
+--      scintillua.styles.list.priority.g_z=fore:$(scintillua.colors.grey),bold
```

Comment:
You should be able to use `scintillua.styles.list.priority=fore:$(scintillua.colors.grey),bold` followed by colors for `scintillua.styles.list.priority.[a-f]`. You don't actually have to define styles for g-z if you don't want to.

Reply:
ahhh, noice!